msprime/mutations.py

```diff
@@ -446,8 +446,10 @@ class TPM(MicrosatMutationModel):
     """
 
     def __init__(self, *, p=None, m=None, lo=None, hi=None, root_distribution=None):
```

Comment:
Simpler here to just take out the ``=None`` bit for these parameters. They become mandatory keyword arguments then.

Reply:
will odo